charts/cloudzero-agent/CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.0.12]
```

Comment:
I do think we're on 0.0.13 technically, we may just be missing the notes from .12. I think the eventual goal plan is to have github actions update this, but for now, would you mind bumping this to 0.13 and adding the [release notes](https://github.com/Cloudzero/cloudzero-charts/releases/tag/0.0.12) for .12 here?